cheatsheets/Session_Management_Cheat_Sheet.md

```diff
@@ -163,6 +163,49 @@ Typically, session management capabilities to track users after authentication m
 - Ensure entire cookie should be encrypted if sensitive data is persisted in the cookie
 - Define all cookies being used by the application, their name and why they are needed
 
+# HTML5 Web Storage API
+
+The Web Hypertext Application Technology Working Group (WHATWG) describes the HTML5 Web Storage APIs, localStorage and sessionStorage, as mechanisms for storing name-value pairs client-side.
+Unlike HTTP cookies, the contents of localStorage and sessionStorage are not automatically shared within requests or responses by the browser and are used for storing data client-side.
+
+## The localStorage API
+
+### Scope
+
+Data stored using the localStorage API is accessible by pages which are loaded from the same protocol, port and host.
+This provides similar access to this data as would be achieved by using the "secure" flag on a cookie, meaning that data stored from https could not be retrieved via http.
+
+### Duration
+
+Data stored using the localStorage API is persisted across browsing sessions, meaning that this data is susceptible to access by other system users. 
```

Comment:
It's the same for `sessionStorage` during the time when the tab is opened. 

So, it's important to explictly mention that data stored by the browser are not ciphered when they are stored for both `sessionStorage` and `localStorage`.

Reply:
It was just a remark from me about the fact that you have the same issue from someone having access to the machine file system whatever the kind of storage used. The only difference is the timeframe in which the data is stored.

Reply:
For the mentioned point, I believe there needs to be another section for it. Keep things well organized 😄 I am not against it.

Reply:
100% agree

Reply:
Raised in a separate heading regarding Offline Access